src/commands/compareReports.js

```diff
@@ -48,6 +48,10 @@ export default async function compareReports(
     typeof compareThreshold === 'number' || dryRun,
   );
   if (typeof compareThreshold !== 'number' || isAsync) {
+
+  if(!isAsync && afterSyncComparison && typeof afterSyncComparison === "function") {
+    afterSyncComparison(firstCompareResult);
```

Comment:
I suggest we `await` the results so that you can run an asynchronous function.

Reply:
Added ;)